### PR TITLE
Trying usual deploy to pypi

### PR DESCRIPTION
**Pull request by emmlejeail**
trying to go around the problem using private settings to set the password in travis instead of encryption in the repo

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,15 +24,9 @@ script:
...
-  provider: pypi
-  user: ttnci
-  password:
-      secure: E7w/gsi1hf7SjkmDX1NV6cyXqG/bS/x+ikcNY79Nh2X8cehVkxa75aglEh8b0SeRSRocgSmF2sqsQvLNMqGQ0T4iCLhV0W7+HVndyOUlNZabMafUW2+bFde18ZahnP6miO+EEliKYoVkDUmkkFmPva9AM741MNge4oHX24dDYXmzTkP4vsyiAa0Y1krdk9gAkAp4ZbLVoJKRKCcq4Vehvk3HVqKptnmlAM8GqT6oYNHrr1ZPNb9TAcGlscHHvsjBlKflqrXaNRn4CA7p4pX2x8cLpIq32EPkRIdc2wohe2/ZarJ5jKvOXnSSNSbSh7OBWIm5bDI8StR4UOeEVyy5Bo/1juSPblLTTjEbV5Bg34vgvWgYeNgYrKynXw8CzuVASIRprB0eUqsWB0jg/S16xtwkTE92SOi8418GL/Atae0nNNS3cmFfrwBNtVx6GsxYdVWzpsIN31fB4R0Wg+j5gxUZEmQ9fHK4MHBKNBSi/Xhph9cjs6Yyu237Gze86jROaoA+jQGrhaJP4WqOTc6dgC35ZZW2btlKUQ+EZMbbJh2KlhEZylgN/P+7lw2V+zHLicfCUrivjEJI2P/fqaQs7YNfXkwOSjp2x0sJRuPHdEnYgsyM7ETMOHCnDaC6Kyob1XamFgxLVQ4C5S0f0HusYETeTP9NU5ZXgKQ
-  branch: master
-  tags: true
+  - pip install twine
+  - python setup.py sdist bdist_wheel
+  - |
+    if [[ $TRAVIS_TAG ]]; then
```
Comment (egourlao) [inline .travis.yml]: Can you link to the documentation regarding this? e.g. in a comment

**Review by egourlao — CHANGES_REQUESTED** [review]

**Comment by egourlao** [comment]
Before circumventing this with a non-standard solution, I'd rather make more efforts to figure out why the standard Travis solution is not working. Look at how other libraries using Travis to push are doing this - if this doesn't work, try with a dummy package (as an open-source repository on your own GitHub for example). Please let us know what you find.

**Comment by emmlejeail** [comment]
I think the problem is coming from the special characters used in the password (generated by Keybase) many issues with the same error mention making sure to have a password without those characters to encrypt the key
However, there is no way to verify the correct encryption by travis
I'll keep making tests on a dummy package and will follow up

**Comment by egourlao** [comment]
I remember reading that Travis does not escape characters from `bash` execution. If this is the case, you could either add escape characters, or change the account password for one that does not have special characters (as long as it is long enough).

**Comment by emmlejeail** [comment]
It turns out I'm perfectly able to upload something on pypi with my personal account (which key doesn't have any special characters) through Travis
[here is the dummy package](https://github.com/emmlejeail/dummy-package)

**Comment by emmlejeail** [comment]
I already tried escaping characters and it doesn't seemed to work the previous pull requests
I'm going to change the password for the pypi account then
@dotpy3

**Review by egourlao — CHANGES_REQUESTED** [review]
As discussed on Slack, going back to native Travis uploading.

**Pull request merged**